content/en/developers/marketplace.md

```diff
@@ -172,15 +180,19 @@ The ddev commands in the previous section generate a skeleton of folders and fil
 * You can create dashboards and monitors directly in your sandbox account and export them as JSON files. 
 * See our [Dashboarding Best Practices][26] document for details.
 
-#### Logo
+#### Logos
 
-* Consists of an SVG, which our DesignOps team implements throughout the Datadog app for both light and dark modes. 
+* Consists of at least one SVG, which our DesignOps team implements throughout the Datadog app for both light and dark modes. Logo SVG files can be added to the `assets` directory, or you can place them in a `logos` sub-directory under `assets`.
 * **Note:** Technology Partners are responsible for the licensing of the logos they submit.  
 
 #### Changelog.md
 
 * Captures release notes and version information, and displays this in the “Release Notes” tab of your tile. Add release notes in descending order (latest version at the top).
 
+#### Codeowners
+
+* Lives under the shared .github directory and defines the individuals or teams that are responsible for code in the repository. See https://help.github.com/articles/about-codeowners/ for help with syntax.
```

Comment:
```suggestion
* Located in the shared `.github` directory and defines the individuals or teams that are responsible for code in the repository. See [Github's documentation][29] for help with syntax.
```

And add this at the very end of the file:
`[29]: https://help.github.com/articles/about-codeowners/`

Reply:
Thanks for updating the link reference. 

Could we use "Lives in" instead of "Located in"? I was trying to keep some consistency with the verb tense for each of these sections (e.g., "Consists," "Captures," "Lives"). Also, this allows "Lives" to share the same tense as "defines" later in the line. I'll accept and then add a new commit for now.